engine/uk.co.spudsoft.birt.emitters.excel/src/uk/co/spudsoft/birt/emitters/excel/StyleManagerUtils.java

```diff
@@ -786,11 +787,12 @@ protected int getRichTextRunIndexForStart(List<RichTextRun> richTextRuns, int st
 	 * @param defaultFont  The font to be used prior to the first RichTextRun.
 	 * @param widthMM      The width of the output.
 	 * @param richTextRuns The list of RichTextRuns to be applied to the string
-	 * @return The heigh, in points, of a box big enough to contain the formatted
+	 * @param wrap         The text use wrapped style
```

Comment:
The comment for wrap is not clear to me (and grammar errors)

Reply:
I will change it.